cassandane/Cassandane/JMAPTester.pm

```diff
@@ -42,7 +42,7 @@ sub CallMethods {
         ($using ? (using => $using) : ()),
     });
 
-    return $res->as_stripped_triples;
+    return $res->assert_successful->as_stripped_triples;
 }
```

Comment:
Nit on this commit message: "and will instead do provide a" "instead do provide a" seems like a typo